docs/pages/CodeAnalysis.md

```diff
@@ -0,0 +1,227 @@
+# Code Analysis
+
+During the build process, UI5 Tooling executes a static code analysis of your project. In the following sections the term "simple literal" is used for the following language features:
+
+- string literals
+- numeric literals
+- Boolean literals
```

Comment:
Boolean is still upper case?

Reply:
It's uppercase in the [SAP API Style Guide](https://help.sap.com/docs/SNG_4_APIDOC/53e39c8b7c924c28a2575be50bc09786/259573468585469f97651cfdfe0e4672.html), in SAPterm / [Terminology Search App](https://lpxtest-approuter-term.cfapps.sap.hana.ondemand.com/ui/), and in our very own internal [UA guidelines for standard formulations in the API Reference](https://workzone.one.int.sap/site#workzone-home&/wiki/show/QXEPJkMTXgPzsAcuGP8pYb).

Reply:
but I see no reason why boolean would be different from string or numeric?

Reply:
Oh, I guess it's because Boolean is named after George Boole, so it's rated as a name. From a technical point of view, I can't understand this. The term is long established and no longer associated with a person. JavaScript and TypeScript both use the lower case name, so for me this is the closest association in our context. 

But okay, we should stick to the guides.

Maybe type string is also named after [George String](https://www.discogs.com/de/artist/1613301-Georg-String) 🤷‍♂️

Reply:
Because it's derived from a surname of a person - the mathematician George Boole. See, for example, 'Newtonian mechanics', 'Hamiltonian operator', 'Euclidian geometry', ... all these are uppercase as well.